nodes/KlickTipp/actions/field/get.operaton.ts

```diff
@@ -0,0 +1,46 @@
+import type { IExecuteFunctions, INodeProperties } from 'n8n-workflow';
```

Comment:
I suppose the file name should be get.operation.ts right?

Reply:
Fixed